telegram_packs/other.txt

```diff
@@ -0,0 +1,88 @@
+other/biFrog|🏳️
+other/croakaColaFrogCan|🧃
+other/doomFrog|🔫
+other/dumbFrog|🤪
+other/eatenFrog|🪦
+other/fingeringFrog|👉
+other/frog2b|🤖
+other/frogBag|💰
+other/frogBatsignal|🌃
+other/frogBeerCan|🍺
+other/frogBeer|🍻🍺
+other/frogBender|🤖
+other/frogBlanket|🛌
+other/frogBlunt|🚬
+other/frogCatCostume|🐱
+other/frogCat|🐱
+other/frogChampagne|🍾
+other/frogChronoTrigger|🐲
+other/frogCoins|🪙
+other/frogDogCostume|🐶
+other/frogDonut|🍩
+other/frogEmil|💀
+other/frogFamilyTkg|👪
+other/frogFlag|🏳️
+other/frogFlash|⚡
+other/frogGun|🔫
+other/frogHacker|🖥️
+other/frogHipster|🤓
+other/frogJester|🃏
+other/frogKaiser|👑
+other/frogKing|👑
+other/frogKirby|🍅
+other/frogMerkel|#️⃣
```

Comment:
?

Reply:
any better suggestions for the Merkel-Raute?

Reply:
the confusion was both for kirby and merkel

For merkel, maybe :judge: ? but now that you said it out loud, i get it now.

The tomato kirby still is a tad vague tho. :purple_circle: maybe, but that one is purple instead of pink.

Reply:
In Kirby there is the tomato item that fills up your health. But yeah there is no good fit... will think about your purple ball thing.